src/darker/black_diff.py

```diff
@@ -64,6 +64,12 @@ class BlackModeAttributes(TypedDict, total=False):
     is_pyi: bool
 
 
+BLACK_ARG_CONVERSIONS: Dict[str, Callable[[str], Union[int, bool]]] = {
+    "line_length": int,
+    "skip_string_normalization": lambda value: value == "True",
+}
+
+
 @lru_cache(maxsize=1)
 def read_black_config(src: Path, value: Optional[str]) -> BlackArgs:
```

Comment:
I'd propose to rewrite this method using https://github.com/akaihola/darker/pull/62/commits/7bbbf74b761d0ba9163894fcd5c0723e03b198fa#diff-c021797c036b1ba8d6aac676393ca67a

There is no need in explicit type casting for Black args and also we'll finally get rid of Click

Reply:
Thanks @Mystic-Mirage, I cherry picked your fix.